examples/ConfigurableFirmata/ConfigurableFirmata.ino

```diff
@@ -137,6 +137,8 @@ void systemResetCallback()
 		else if (IS_PIN_DIGITAL(i)) 
 		{
 			Firmata.setPinMode(i, PIN_MODE_OUTPUT);
+// Uncomment following line, if you want to start the board at HIGH output state on powering up:
+//			digitalWrite(pin, HIGH);
```

Comment:
You should also call `Firmata.setPinState()` to make sure the internal pin state cache is updated.

Reply:
Good idea! Thanks for the tip! 
Also I see I have made 2 big mistakes:  
- Forgot the `Firmata.` part, 
- wrote `pin,` instead of `i,`